packages/ui/src/modal/Modal.tsx

```diff
@@ -83,6 +83,7 @@ const Modal: FunctionComponent<ModalProps> = ({
             <div
                 className={classNames('modal-body', additionalBodyClassName)}
                 data-test="modal-body"
+                data-testid="modal-body"
```

Comment:
Can we keep using `data-test` here and the file above?

Reply:
We previously discussed setting `data-test` globally in the test setup.

Reply:
There is

Reply:
It can be configured like this.
https://testing-library.com/docs/dom-testing-library/api-configuration/#introduction

Reply:
Right now, the change is at https://github.com/bigcommerce/checkout-js/pull/1295/files#diff-deeb46ca9b5c701d23c6d445b9d3d51379bdea6649ca3b25661a79a289d16f5d. It is going to be merged.